src/WebJobs.Script/Host/PrimaryHostCoordinator.cs

```diff
@@ -227,7 +227,7 @@ public void Dispose()
 
         public Task StartAsync(CancellationToken cancellationToken)
         {
-            _timer.Change(TimeSpan.Zero, _leaseRetryInterval);
+            _timer.Change(_leaseRetryInterval, _leaseRetryInterval);
```

Comment:
Would be good to have a comment here detailing why we are delaying this. This will be easy to regress

Reply:
Will add a comment and have it in with my next round of changes.